packages/http-json-body-parser/index.d.ts

```diff
@@ -8,6 +8,6 @@ interface Options {
 
 export type VersionedApiGatewayEvent = APIGatewayEvent | APIGatewayProxyEventV2
```

Comment:
Should `| ALBEvent` go here instead?

Reply:
I was thinking about that too, the only reason I added it like I did instead is that ALBEvent isn't technically an ApiGatewayEvent. Maybe we could rename `VersionedApiGatewayEvent`:
 `export type RequestEvent = APIGatewayEvent | APIGatewayProxyEventV2 | ALBEvent` 
Or something along those lines. Let me know what you think!
`

Reply:
`RequestEvent ` Sounds good to me